source/classes.tex

```diff
@@ -4643,8 +4643,9 @@
 member function, the reference is ill-formed if the
 left operand (considered as a pointer in the
 ``\tcode{.}''
-operator case) cannot be implicitly converted to a
-pointer to the naming class of the right operand.
+operator case), with the cv-qualification on the pointed to class discarded,
+cannot be implicitly converted to a pointer to
+the cv-unqualified version of the naming class of the right operand.
```

Comment:
We can't, because "convert type T to type U" isn't a defined thing. We only have "convert expression E to type T". (This is because the validity of the conversion depends on the value category of the source, not just its type.)